orchagent/orchdaemon.cpp

```diff
@@ -904,7 +904,13 @@ void OrchDaemon::start()
                         for (auto& pair: gPortsOrch->getAllPorts())
                         {
                             auto& port = pair.second;
-                            gPortsOrch->setBridgePortLearningFDB(port, SAI_BRIDGE_PORT_FDB_LEARNING_MODE_DISABLE);
+                            // TODO: how to support 1D bridge?
```

Comment:
@DennisChiuEC is this the single place where `setBridgePortLearningFDB` is being used? If not, that might cause some issues, since behavior has been changed

Reply:
Yes, this is the single place where setBridgePortLearningFDB() is called.